skins/cat17/address-form/src/store.ts

```diff
@@ -73,15 +76,28 @@ export const getters: GetterTree<AddressState, any> = {
 	}
 }
 
+function allFieldsAreEmpty ( formFields: FormData, ignoreFields: string[] ): boolean {
```

Comment:
I consider this more of a private utility function. As long as no other code outside of this module needs it, I'd go with the "private by default" rule, to keep the public API of the store small.

Reply:
> to keep the public API of the store small.

I don't understand this concern. This store is not big and this address update folder is the only place we use vuex in `cat17` and hopefully 🤞 that won't change in the future.
I really don't like that this is just a "free flying" function in the middle of the store. I think it pollutes it. 
If you don't like it as a getter, how about putting it in the `Helper` mixin?

Reply:
It's a leading practice  - Just like we make all methods private by default if they aren't needed by other code. The benefit of making code private is that you can refactor, deprecate and optimize at will and won't break and code that depends on your class. If something is public there will always be some reluctance to change, because it might break something.

A getter would be not the right choice here, because getters should return some state-dependent values. `allFieldsAreEmpty` is a pure function  that does not get any store state.

I've moved the function to the helpers.

Reply:
Right, it doesn't use `state` here. In regards to porting this to `laika` that function will likely use state, because there we keep the values in the store as well, but that's another story.
Therefor this being in Helper will be the best choice imo. Thanks for moving it.
On a side note, the comparison to private functions is not completely fair because that concept does not exist in javascript. Defining it in the scope of the store but not attaching it to any object, does not make the function private as in the php sense, which I think you meant.

Reply:
With typescript/ES6 it's semi-private as the function itself is not exported in the module. That's the best JavaScript can do atm 🤷‍♂